research/object_detection/core/post_processing.py

```diff
@@ -496,3 +496,73 @@ def _single_image_nms_fn(args):
     return (batch_nmsed_boxes, batch_nmsed_scores, batch_nmsed_classes,
             batch_nmsed_masks, batch_nmsed_additional_fields,
             batch_num_detections)
+
+def combined_non_max_suppression(boxes,
+                                 scores,
+                                 score_thresh,
+                                 iou_thresh,
+                                 max_size_per_class,
+                                 max_total_size=0,
+                                 clip_window=None,
+                                 change_coordinate_frame=False,
+                                 num_valid_boxes=None,
+                                 masks=None,
+                                 additional_fields=None,
+                                 use_static_shapes=False,
+                                 scope=None):
+  """Multi-class version of non maximum suppression that operates on a batch.
+
+  This op is similar to `multiclass_non_max_suppression` but operates on a batch
+  of boxes and scores. See documentation for `multiclass_non_max_suppression`
```

Comment:
Are these all simply copy/pasted? You should add some comments about that this method really do here.

Reply:
@pkulzc could you PTAL. I just added a better explanation of what the function does.